src/deadline/blender_submitter/addons/deadline_cloud_blender_submitter/blender_utils.py

```diff
@@ -74,6 +74,13 @@ def find_files(project_path, skip_temp=True, skip_nonexistent=True) -> list[Path
         skip_temp: if True, skip all files from any of Blender's potential temp directories.
         skip_nonexistent: if True, skip all files that do not exist. When files are shared across machines, Blender may retain memory of original paths; this ensures that all retrieved paths exist on the local filesystem.
     """
+    import os
+
+    # Get allowlist from environment variable (semicolon-separated on Windows, colon-separated on Unix)
+    allowlist_env = os.environ.get("BLENDER_TEMP_ALLOWLIST", "")
+    separator = ";" if os.name == "nt" else ":"
+    temp_allowlist = [Path(p.strip()) for p in allowlist_env.split(separator) if p.strip()]
```

Comment:
nit: we can use [`os.pathsep`](https://docs.python.org/3/library/os.html#os.pathsep) here

```suggestion
    temp_allowlist = [Path(p.strip()) for p in allowlist_env.split(os.pathsep) if p.strip()]
```

Reply:
good call, made this change.